documentation/modules/new-migrating-virtual-machines-cli.adoc

```diff
@@ -0,0 +1,1046 @@
+    name: <secret> <2>
+    namespace: <namespace>
+EOF
+----
+<1>  where: `nfs_server` is an IP or hostname of the server where the share was created and `nfs_path` is the path on the server where the OVA files are stored.
+<2> Specify the name of provider `Secret` CR.
+endif::[]
+
+ifdef::ostack[]
+[source,yaml,subs="attributes+"]
+----
+$ cat << EOF | {oc} apply -f -
+apiVersion: forklift.konveyor.io/v1beta1
+kind: Provider
+metadata:
+  name: <source_provider>
+  namespace: <namespace>
+spec:
+  type: openstack
+  url: <api_end_point> <1>
+  secret:
+    name: <secret> <2>
+    namespace: <namespace>
+EOF
+----
+endif::[]
+
+ifdef::cnv[]
+[source,yaml,subs="attributes+"]
+----
+$ cat << EOF | {oc} apply -f -
+apiVersion: forklift.konveyor.io/v1beta1
+kind: Provider
+metadata:
+  name: <source_provider>
+  namespace: <namespace>
+spec:
+  type: openshift
+  url: <api_end_point> <1>
+  secret:
+    name: <secret> <2>
+    namespace: <namespace>
+EOF
+----
+<1> Specify the URL of the endpoint of the API server.
+<2> Specify the name of provider `Secret` CR.
+endif::[]
+
+ifdef::vmware[]
+[start=3]
+. Create a `Host` manifest:
++
+[source,yaml,subs="attributes+"]
+----
+$ cat << EOF | {oc} apply -f -
+apiVersion: forklift.konveyor.io/v1beta1
+kind: Host
+metadata:
+  name: <vmware_host>
+  namespace: <namespace>
+spec:
+  provider:
+    namespace: <namespace>
+    name: <source_provider> <1>
+  id: <source_host_mor> <2>
+  ipAddress: <source_network_ip> <3>
+EOF
+----
+<1> Specify the name of the VMware vSphere `Provider` CR.
+<2> Specify the managed object reference (MOR) of the VMware vSphere host.
+<3> Specify the IP address of the VMware vSphere migration network.
+
+[start=4]
+. Create a `NetworkMap` manifest to map the source and destination networks:
++
+[source,yaml,subs="attributes+"]
+----
+$  cat << EOF | {oc} apply -f -
+apiVersion: forklift.konveyor.io/v1beta1
+kind: NetworkMap
+metadata:
+  name: <network_map>
+  namespace: <namespace>
+spec:
+  map:
+    - destination:
+        name: <network_name>
+        type: pod <1>
+      source: <2>
+        id: <source_network_id>
+        name: <source_network_name>
+    - destination:
+        name: <network_attachment_definition> <3>
+        namespace: <network_attachment_definition_namespace> <4>
+        type: multus
+      source: <5>
+        id: <source_network_id>
+        name: <source_network_name>
+  provider:
+    source:
+      name: <source_provider>
+      namespace: <namespace>
+    destination:
+      name: <destination_provider>
+      namespace: <namespace>
+EOF
+----
+<1> Allowed values are `pod` and `multus`.
+<2> You can use either the `id` _or_ the `name` parameter to specify the source network. For `id`, specify the VMware vSphere network MOR.
+<3> Specify a network attachment definition for each additional {virt} network.
+<4> Required only when `type` is `multus`. Specify the namespace of the {virt} network attachment definition.
+<5> Same as `source` above.
```

Comment:
this sounds like the 'id' / 'name' property should have the same value as in the mapping above - how about dropping `<5>`? (same for the other provider below)